tests/src/JIT/Performance/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,38 @@
+@setlocal
+@echo off 
+
+set ARCH=%1
+set BUILD=%2
+
+if /I "%ARCH%" == "" set ARCH=x64
+if /I "%BUILD%" == "" set BUILD=Release
+
+rem *** set this appropriately for enlistment you are running benchmarks in
+if /I NOT "%CORECLR_ROOT%" == "" goto over
```

Comment:
%~dp0 Gets the directory of the running batch file, so you can use that + ..\..\ etc. to get the root of the repo.  We use something similar in the msbuild project files.